Documentation/how-to-run-tests.md

```diff
@@ -84,9 +89,75 @@ REGEDIT4
 ```
 
 ### Filtering Tests
+
 If you know a test is failing for a good reason or you want a clean baseline, please use ```corert\tests\KnownGood.CoreCLR.issues.targets``` to weed out bad tests or infrastructure errors until we fix them.
 
 ### Test Logs
+
 When the tests finish execution, the log location will be written out and should be in bin\Logs:
 
 **Example:** ```corert\bin\Logs\TestRun_Windows_NT__x64__debug.html```
+
+## External tests - CoreFX
+
+Similarly to the CoreCLR tests, when runtest.cmd is passed the /corefx switch, the harness will download the CoreFX project's test suite, compile them to native with the CoreRT compiler, and run them.
+
+### How to run
+
+To run CoreFX tests on CoreRT, make sure that `build.cmd` has been run at least once in the configuration you'd like to test (i.e. `Debug` or `Release`), open a new console window and from the repo root execute the following:
+
+On Windows:
+
+```
+tests\runtest.cmd /corefx
+```
+
+On Linux / macOS:
+
+```
+tests/runtest.sh -corefx
+```
+
+The tests assemblies to run are defined in `TopN.CoreFX.issues.json` with their respectively excluded test methods, classes or namespaces.
+
+### Reproducing test failures
+
+If you need to reproduce a failing test, from the repo root navigate to ```test_downloaded\CoreFX``` and then to the folder of the failing test - each test suite is located in its own folder. From the repo root run the following:
```

Comment:
Is this really executed from the repo root? ("From the repo root run the following")

Reply:
Actually good point - "from root navigate to..." is a bit clunky.